packages/fxa-auth-client/lib/client.ts

```diff
@@ -2288,7 +2295,20 @@ export default class AuthClient {
     headers?: Headers
   ) {
     return this.sessionPost(
-      '/recovery_phone/available',
+      '/recovery_phone/confirm',
```

Comment:
@LZoog You'll need this update when you work on the step to confirm code during recovery phone setup